src/commands/build.ts

```diff
@@ -276,63 +284,69 @@ export async function command(commandOptions: CommandOptions) {
           continue;
         }
 
+        allBuiltFromFiles.add(fileLoc);
         if (path.extname(outPath) === '.js') {
           if (resources?.css) {
             const cssOutPath = outPath.replace(/.js$/, '.css');
             await fs.mkdir(path.dirname(cssOutPath), {recursive: true});
             await fs.writeFile(cssOutPath, resources.css);
             code = `import './${path.basename(cssOutPath)}';\n` + code;
           }
-          const resolveImportSpecifier = createImportResolver({
-            fileLoc,
-            webModulesPath,
-            dependencyImportMap,
-            isDev: false,
-            isBundled,
-            config,
-          });
-          code = await transformEsmImports(code, (spec) => {
-            // Try to resolve the specifier to a known URL in the project
-            const resolvedImportUrl = resolveImportSpecifier(spec);
-            if (resolvedImportUrl) {
-              // We treat ".proxy.js" files special: we need to make sure that they exist on disk
-              // in the final build, so we mark them to be written to disk at the next step.
-              if (resolvedImportUrl.endsWith('.proxy.js')) {
-                allProxiedFiles.add(
-                  resolvedImportUrl.startsWith('/')
-                    ? path.resolve(cwd, spec)
-                    : path.resolve(path.dirname(outPath), spec),
-                );
-              }
-              return resolvedImportUrl;
-            }
-            // If that fails, return a placeholder import and attempt to resolve.
-            let [missingPackageName, ...deepPackagePathParts] = spec.split('/');
-            if (missingPackageName.startsWith('@')) {
-              missingPackageName += '/' + deepPackagePathParts.shift();
-            }
-            messageBus.emit('MISSING_WEB_MODULE', {
-              id: fileLoc,
-              data: {
-                spec: spec,
-                pkgName: missingPackageName,
-              },
-            });
-            // Sort of lazy, but we expect "MISSING_WEB_MODULE" to exit the build with an error.
-            // So, just return the original import here since it will never be seen.
-            return spec;
-          });
           code = wrapImportMeta({code, env: true, hmr: false, config});
+          allBuiltJsFiles.push([outPath, code, fileLoc]);
```

Comment:
With the tuple comment above, this could become:

```diff
-allBuiltJsFiles.push([outPath, code, fileLoc]);
+allBuiltJsFiles.push({outPath, code, fileLoc});
```

Less margin for error, more clarity when adding.